src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/Data/Json/GetRules/GetRules_RuleTypeInheritedAsSubunitViaKeyroleRequest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parentPartyId": 0,
```

Comment:
parentPartyId: 0
Should not be able to result in InheritedAsSubunit scenario neither directly nor via KeyRole?
Haven't verified expected outcome of the test but name of the JSON file suggests this is the scenario

Reply:
Fixed SetRuleType so it sets the correct ruletype, and updated the unit tests